pylint/lint/parallel.py

```diff
@@ -23,19 +23,23 @@
 except ImportError:
     multiprocessing = None  # type: ignore[assignment]
```

Comment:
This is one use of this still, it is used to get the id of the process. Can we do that with `concurrent`?